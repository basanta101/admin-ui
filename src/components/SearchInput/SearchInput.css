.searchInput {
  width: calc(100% - 34px);
  height: 30px;
  border: 1px solid #aaa;
  border-radius: 4px;
  padding: 0 16px;
  margin-bottom: 4px;
}
