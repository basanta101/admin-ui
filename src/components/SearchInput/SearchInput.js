import './SearchInput.css';

const SearchInput = (props) => {
    return (
        <input type='text' className='searchInput' {...props}/>
    )
}

export default SearchInput;