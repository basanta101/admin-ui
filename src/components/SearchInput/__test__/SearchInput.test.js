import renderer from "react-test-renderer";
import SearchInput from "../SearchInput";

it("renders a snapshot of Pagination Component", () => {
  const tree = renderer.create(<SearchInput />).toJSON();
  expect(tree).toMatchSnapshot();
});
