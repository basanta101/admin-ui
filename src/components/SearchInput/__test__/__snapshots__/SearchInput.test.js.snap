// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of Pagination Component 1`] = `
<input
  className="searchInput"
  type="text"
/>
`;
